app/public/css/styles.css

```diff
@@ -553,6 +558,8 @@ button[data-balloon] {
 
 .alert-content {
   text-align: left;
+  max-height: 300px;
+  overflow: hidden;
```

Comment:
overflow: scroll/auto; would work better with long exceptions

Reply:
I was inspired by myEtherwallet, anyway I tried the "scroll" option, but the modal looks visually much worse with two scrollbars so I changed it back.